WebAPI/LearningHub.Nhs.Services/ResourceService.cs

```diff
@@ -4941,7 +4941,7 @@ private async Task<int> CreateNewDraftVersionForResource(int resourceId, Resourc
 
         private async Task<int> CreateNewResource(ResourceTypeEnum resourceType, string title, int userId, AssessmentTypeEnum? assessmentType)
         {
-            var resourceVersionId = await this.resourceRepository.CreateResourceAsync(resourceType, title, string.Empty, userId);
+            var resourceVersionId = await this.resourceRepository.CreateResourceAsync(resourceType, title, string.Empty, 1, userId);
```

Comment:
It looks like this method is called when duplicating a resource (a Case). As the new resource is stored in the same folder as the original resource the PrimaryCatalogueId should also be the same. i.e. not set to 1